pom.xml

```diff
@@ -62,7 +62,7 @@
                 <version>${jitsi.utils.version}</version>
             </dependency>
             <dependency>
-                <groupId>com.github.jitsi</groupId>
+                <groupId>${project.groupId}</groupId>
```

Comment:
Hm, in this pom you have a direct dependency on jitsi-metaconfig, whereas in jmt you don't (you just let the jicoco dependency pull it in recursively).

Either approach is probably okay, but the inconsistency seems odd?

Reply:
Nice catch...both JVB and JMT use functions from metaconfig directly, so I think they should be explicitly set the dependency...I must have just missed it in JMT.